ClaudiaHernandezGarcia/springBootProject/src/main/java/com/globant/springBootProject/controller/HackerrankController.java

```diff
@@ -0,0 +1,49 @@
+package com.globant.springBootProject.controller;
+
+import com.globant.springBootProject.dto.AnagramsDTO;
+import com.globant.springBootProject.dto.PatternSyntaxCheckerDTO;
+import com.globant.springBootProject.dto.ResponseDTO;
+import com.globant.springBootProject.dto.StringReverseDTO;
+import com.globant.springBootProject.service.AnagramsService;
+import com.globant.springBootProject.service.PatternSyntaxCheckerService;
+import com.globant.springBootProject.service.StringReverseService;
+import com.globant.springBootProject.utils.Constants;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("hackerrank")
+public class HackerrankController {
+    @Autowired
+    StringReverseService stringReverseService;
+
+    @Autowired
+    AnagramsService anagramsService;
+
+    @Autowired
+    PatternSyntaxCheckerService patternSyntaxCheckerService;
+
+    @PostMapping(path = "stringReverse", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<ResponseDTO<StringReverseDTO>> getIsPalindrome(@RequestBody StringReverseDTO stringReverseDTO) {
```

Comment:
getIs.... ??? don't think is a good name for a method right?

Reply:
method names still getIsSomething.... not a good name